src/routes/docs/tutorials/android/step-2/+page.markdoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: tutorial
+title: Initialize project
+description: Learn to build an Android app with no backend code using an Appwrite backend.
+step: 2
+difficulty: beginner
+---
+
+# Initialize Android Studio project {% #initialize-android-studio-project %}
+
+1. Fire up your Android Studio
+
+2. Go to **File / New / New Project**
+
+3. Select **Empty Activity**
+
+![Empty Activity](/images/docs/tutorials/recipewrite-empty-activity.png)
+
+4. Name your project **Recipewrite**, select Java language, and minimum SDK would be **23**
+
+![New Project](/images/docs/tutorials/recipewrite-new-project.png)
+
+# Add dependencies {% #add-dependencies %}
+
+1. Open **app/build.gradle** file
+
+2. Find `dependencies` block, and add the following dependencies along with existing ones
+    ```
+    implementation 'io.appwrite:sdk-for-android:4.0.0'
```

Comment:
This should be double quotes " with ()

Reply:
Not sure what you mean here, can you please elaborate? Sorry.